mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/content/Language.properties

```diff
@@ -19,9 +21,9 @@ problem=Problem
 profile-page-is-not-updated=Profile page is not updated
 profile-problem=Profile problem
 sample-task=Sample Task
-success=Form successfully submitted!
+success=Form successfully submitted
```

Comment:
OK. Here we have almost same case as with error key word + violation of the rule that says key must be equal to sentence up to first dot and spaces are replaced with dashes.
Please update this.